package.json

```diff
@@ -14,6 +14,7 @@
     "clients/client"
   ],
   "dependencies": {
+    "mdx-loader": "^2.2.1",
```

Comment:
This dependency is required in the root repo if the generated docs are not located in the `ui/`. I currently have `mdx-loader` defined both in `package.json` and `ui/package.json`...

Reply:
This is also why netlify is failing since it's not doing `yarn install` in the root directory. Thoughts on removing `generated/docs` and `ui/src/docs` and instead have everything under `ui/docs` (i.e.,  `ui/docs/static` and `ui/docs/generated`)?

Reply:
That sounds 110% fine to me.

Reply:
👍